lib/vtls/openssl.c

```diff
@@ -90,6 +90,10 @@
 #include <openssl/engine.h>
 #endif
 
+#if (OPENSSL_VERSION_NUMBER >= 0x10100000L) /* 1.1.0 or later */
```

Comment:
```suggestion
#if (OPENSSL_VERSION_NUMBER >= 0x10101000L) /* 1.1.1 or later */
```
https://github.com/openssl/openssl/issues/13548#issuecomment-734766906

Reply:
Oops...